test-openvpn.ps1

```diff
@@ -35,12 +35,16 @@ Function Verify-Path ([string]$mypath) {
     }
 }
 
+$exe_path = (Get-ItemProperty 'HKLM:\SOFTWARE\OpenVPN' -Name 'exe_path').exe_path
+
 if (! $Openvpn) {
-    $Openvpn = "C:\Program Files\OpenVPN\bin\openvpn.exe"
+    $Openvpn = $exe_path
 }
 
 if (! $Gui) {
-    $Gui = "C:\Program Files\OpenVPN\bin\openvpn-gui.exe"
+    # Assume openvpn-gui.exe is in the same directory as openvpn.exe
```

Comment:
there's separate registry value for openvpn-gui.exe, shouldn't we take it instead ?